implementation/src/main/module/module-info.java

```diff
@@ -0,0 +1,27 @@
+module io.smallrye.mutiny {
```

Comment:
Isn't this file being generated so shall it be in Git?

Reply:
No, unfortunately, the generation failed on Java 8 (requires Java 9). So we need to have the content in git and update it every time we do some change (new services or new packages).